test/await_workflow_completion.py

```diff
@@ -51,18 +51,14 @@ def get_statuses(names, ids, cromwell_url, user, pw):
             print('{0} {1} workflow {2}: {3}'.format(timestamp, name, id, status))
     return statuses
 
-def extract_credentials(secrets_file):
-    with open(secrets_file) as f:
-        secrets = json.load(f)
-        return secrets['cromwell_user'], secrets['cromwell_password']
-
 if __name__ == '__main__':
     parser = argparse.ArgumentParser()
     parser.add_argument('--workflow_ids')
     parser.add_argument('--workflow_names')
     parser.add_argument('--cromwell_url')
-    parser.add_argument('--secrets_file')
+    parser.add_argument('--cromwell_user', default=os.environ.get('cromwell_user', None))
```

Comment:
I eliminated the secrets file in this PR, which seems like an unnecessary complication now. That means we have to pass the secrets in via arguments here.